todos-app/frontend/src/components/TaskEditModal.vue

```diff
@@ -32,6 +32,17 @@
 
             <label class="form-label mt-4">Due Date</label>
             <input name="due_datetime" type="datetime-local" class="form-control" :value="task.due_datetime ? task.due_datetime.substring(0, 16) : null">
+
+            <label class="form-label mt-4">Tags</label>
+            <div class="d-flex flex-wrap">
+              <span v-for="(label, i) in labels" :key="i" class="badge rounded-pill badge-outline-secondary m-1">#{{label}}<i type="button" class="bi bi-x" @click="removeLabel(label)"></i></span>
+            </div>
+            <div class="input-group">
+              <input name="label" type="text" class="form-control" placeholder="Max 15 characters without space" v-model="labelInput">
+              <button type="button" class="btn btn-outline-secondary input-group-text" @click="addLabel">
+                <i class="bi bi-plus-circle"></i>
+              </button>
+            </div>
```

Comment:
It seemed like this part is kinda same with the TaskCreateModal.vue part.
Do vue file have something that can group and re-use components like `partial` in rails?(Usually for rails, we will use partial for the whole form and reuse it in `create` and `edit`'s html